### DISCUSSION TITLE
Visible total number of stacked bars in mixed time-series

### DISCUSSION THREAD

**Discussion by LeoDiep** [apache/superset#27165] [Ideas]
Hi Community,
I am using Superset version 2.1 and have a little problem.

For Time-serie bar chart (legacy) we have option to visible the bar value (total value) of stacked components.
![image](https://github.com/apache/superset/assets/99783557/2218a8f1-f1c5-4a63-8b58-e1db464637ad)

Same for the new Time-serie bar chart, which we have more control over the value visibled be total value only or component values.
![image](https://github.com/apache/superset/assets/99783557/d966c096-e496-42e3-8c07-6aa79bfdd82d)

But when it comes to the Mixed time-series chart, there is no option to visible the total number of stacked values of either query A or query B, which make it difficult for user to have a glance at total number of 1 metric.
(Only the Show Value option here make the chart are uneasy to track)
![image](https://github.com/apache/superset/assets/99783557/91a5e8c5-096e-4cfc-ad6e-35fcc8f138a1)

Please help us make the Only Total work on the Mix time-series chart.
Thank you.

P/s: Is this fixed in newly version after v2.1?

**Comment by sfirke** [comment]
It's not fixed in 3.0.0, I just came to report this as a desired feature request.

**Reply by LeoDiep** [reply]
Thank you a lot. Best of wishes

**Comment by LeoDiep** [comment]
Hi everyone, is there any update on this feature missing
